src/System.ComponentModel.TypeConverter/src/System/ComponentModel/WarningException.cs

```diff
@@ -66,7 +68,8 @@ public WarningException(string message, string helpUrl, string helpTopic)
         /// </summary>
         protected WarningException(SerializationInfo info, StreamingContext context) : base(info, context)
         {
-            throw new PlatformNotSupportedException();
+            HelpUrl = (string)info.GetValue("helpUrl", typeof(string));
```

Comment:
>helpUrl [](start = 45, length = 7)

I suppose casing here is like that in order to match Desktop. Should we add a comment or something so that it doesn't get renamed?

Reply:
Casing of the field or the string? The string matches Desktop, correct. The HelpUrl field/property is from the base type Exception.

Reply:
I meant the string, `helpUrl`. I thought you were adding comments in other serializable types so that people won't rename strings since it would cause a compatibility with desktop.

Reply:
Jep I will do that with another cleanup change that Stephen suggested (replacing the assemblyqualifiednames in the typeforwards with consts) in a separate PR after I finished the tests and hopefully merged it into the servicing release.